### ISSUE TITLE
Pass customer's keys to SLM

### ISSUE THREAD

**Issue by jbonnet** [sonata-nfv/son-gkeeper#943]
#890 automatically generates the keys when a user of type `customer` is created, so when an instantiation request is received, the Gatekeeper should colect those keys and pass them to the SLM, like in
```yaml
user_data:
  customer:
    email: ...
    phone: ...
    keys:
      public: ...
      private: ...
```

**Comment by jbonnet** [comment]
Hey @tsoenen, @dang03 (cc @felipevicens )
I think it's done! Please use and let me know if that's ok...

**Comment by jbonnet** [comment]
Closed by PR #944
